charts/cosi/templates/_helpers.tpl

```diff
@@ -178,11 +166,7 @@ Create the full name of provisioner sidecar image from repository and tag
 Create the secret name
```

Comment:
This definition can be removed, and we can default to cosi-config now that we will require it as a prerequisite.